src/beginner/src/ch02-03-chords.md

```diff
@@ -131,4 +130,4 @@ Ppar([melody, prog]).play
 So here you can see we have a new pattern type: ```Ppar```. Ppar allows us to play two independent
```

Comment:
strictly speaking, it is a "pattern class", not a "pattern type". It depends of course if you want to speak in computer science terms or not.